astropy/utils/metadata.py

```diff
@@ -398,20 +397,21 @@ def __get__(self, instance, owner):
         if instance is None:
             return self
         if not hasattr(instance, '_meta'):
-            instance._meta = OrderedDict()
+            instance._meta = dict()
         return instance._meta
 
     def __set__(self, instance, value):
         if value is None:
-            instance._meta = OrderedDict()
-        else:
-            if isinstance(value, Mapping):
-                if self.copy:
-                    instance._meta = deepcopy(value)
-                else:
-                    instance._meta = value
+            instance._meta = dict()
+            return
+
+        if isinstance(value, Mapping):
```

Comment:
This code is the same as the original except OrderedDict => dict and doing a `return` instead of an `else` block.

Reply:
Is this just a note or something I still have to do? I don't quite remember. 😬